packages/composites/rocket-pool/src/config/index.ts

```diff
@@ -0,0 +1,28 @@
+import { Requester, util } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/ea-bootstrap'
+import { ethers } from 'ethers'
+import rethAbi from '../abis/rethAbi.json'
+
+export const NAME = 'ROCKET_POOL'
+
+export const DEFAULT_ENDPOINT = 'reth'
+export const DEFAULT_ETH_USD_PROXY_ADDRESS = '0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419'
+export const RETH_TOKEN_CONTRACT = '0xae78736Cd615f374D3085123A210448E74Fc6393'
+
+export type RocketPoolConfig = Config & {
+  rethContract: ethers.Contract
+}
+
+export const makeConfig = (prefix?: string): RocketPoolConfig => {
+  const ethereumRpcUrl = util.getRequiredEnv('ETHEREUM_RPC_URL')
```

Comment:
The EA endpoint looks like it is set up to be cross-chain. Should this support multiple networks as the `network` input param implies?

```
description: 'Network to query for price feed (EA must have `<network>_RPC_URL` configured).',
```
We have previously used a combo of a dynamic prefix from the `network` + a fallback.
```
// warning: pseudocode
// build a dynamic env variable look up here
  const rpcURL = util.getRequiredEnvWithFallback(
    ENV_RPC_URL,
    [ENV_FALLBACK_RPC_URL],
    prefix,
  )
```

This is probably something we will want to standardize into a utility function.

Reply:
Yeah the idea here is that the EA would support multiple chains for the `ETH/USD` price feed, but the `rETH/ETH` exchange rate comes solely from mainnet (for current implementation).

Reply:
I didn't know how to best capture dynamic env vars in the `schema/env.json`, so I simply referenced the `reference-data-reader` and tried to explain in the README.